webextensions/sidebar/mouse-event-listener.js

```diff
@@ -256,12 +261,25 @@ function onMouseDown(event) {
     promisedMousedownNotified: Promise.resolve()
   };
 
-  mousedown.promisedMousedownNotified = browser.runtime.sendMessage(Object.assign({}, mousedownDetail, {
-    type:     Constants.kNOTIFY_TAB_MOUSEDOWN,
-    window:   mTargetWindow,
-    windowId: mTargetWindow,
-    tabId:    tab && tab.id
-  })).catch(ApiTabs.createErrorHandler());
+  mousedown.promisedMousedownNotified = Promise.all([
+    browser.runtime.sendMessage({type: Constants.kNOTIFY_TAB_MOUSEDOWN })
+      .catch(ApiTabs.createErrorHandler()),
+
+    (async () => {
+      log('Constants.kNOTIFY_TAB_MOUSEDOWN');
+      await Tab.waitUntilTracked(mousedownDetail.tab);
```

Comment:
This check (L270-L273) looks needless anymore. Originally this was introduced for safety about cross-namespace asynchronous operation, and now the `tab` (got with `const tab = EventUtils.getTabFromEvent(event) || EventUtils.getTabFromTabbarEvent(event);`) looks certainly available.